src/Components/Price.tsx

```diff
@@ -1,15 +1,16 @@
-import React, { useContext, useState } from 'react';
+import React, { useContext } from 'react';
 import { HouseDemand, HouseDemandContext } from './HouseDemandContext';
 
-function CalculatePrice({milestones, demand, unitPrice, hasGarden} : HouseDemand): number
+function CalculatePrice({milestones, demand, unitPrice, hasGarden, hasCFO} : HouseDemand): number
 {
     let numItems = demand.total;
     let basePrice = numItems * unitPrice.unitPrice;
     let housePrice = basePrice * (hasGarden ? 2 : 1);
     let drinkBonus = demand.drinks * (milestones.hasDrinkBonus ? 5 : 0);
     let burgerBonus = demand.burgers * (milestones.hasBurgerBonus ? 5 : 0);
     let pizzaBonus = demand.pizza * (milestones.hasPizzaBonus ? 5 : 0);
-    return housePrice + drinkBonus + burgerBonus + pizzaBonus;
+    let total = housePrice + drinkBonus + burgerBonus + pizzaBonus;
+    return hasCFO ? Math.ceil(total * 1.5) : total;
```

Comment:
#3 talks about having the text show the base price and the CFO bonus split out, can we add that in?

Reply:
Fair point, I will add this in the next set of changes when I add waitresses too since if we display CFO separately we presumably want to display waitresses separately.